README.md

```diff
@@ -308,6 +308,7 @@ Additional lists you might find useful:
 - :strawberry: [RowLocker plugin](https://github.com/lorenzo/row-locker) - Exclusive locks for rows in your tables.
 - [Serializeable Data Types plugin](https://github.com/burzum/cakephp-serialize-data-types) - Serialize DB content as JSON or using phps serializing functions.
 - :strawberry: [Muffin/Webservices ORM plugin](https://github.com/usemuffin/webservice) - An ORM like interface for webservices.
+- :strawberry: [Connehito/Master Replica Plugin](https://github.com/Connehito/cakephp-master-replica) - Switch master/replica database connections.
```

Comment:
The plugin name seems to be `Connehito\\CakephpMasterReplica` => `Connehito\\CakephpMasterReplica plugin` as per guidelines.